metrics/coverage_converter/docs/02_coverage_converter_publishing.md

```diff
@@ -0,0 +1,234 @@
+
+`https://github.com/platform-platform/monorepo/releases/latest/download/coverage_converter_macos`
+
+## Using the Coverage Converter tool
+
+Once you've obtained the download link, you can download the Coverage Converter tool using the CLI tools. Let's consider the macOS command to download the Coverage Converter tool:
+
+`curl -o <output> -k <url>`
+
+Where the `<output>` is the file path where you want to download the coverage converter, and the `<url>` is the download URL obtained previously. 
+
+So, when you've got the Coverage Converter executable, you probably, want to convert coverage to be readable by the CI integrations tool. To run the conversion process, you should have the coverage report in one of the supported formats: 
+
+- `LCOV`
+- `Istanbul`
+
+Let's consider the command for converting the `LCOV` format for macOS: 
+
+`./coverage_converter_macos lcov -i <YOUR_COVERAGE_REPORT_FILE> -o <COVERAGE_REPORT_OUTPUT_FILE.json>`
+
+As you can see, the Coverage Converter CLI contains a separate command for each supported coverage format. So, to convert the `Istanbul` coverage report, you should replace the `lcov` command with `istanbul` and so on. See [Coverage Converter design](https://github.com/platform-platform/monorepo/blob/master/metrics/coverage_converter/docs/01_coverage_converter_design.md#cli-design) document for detailed description of the CLI arguments.
+
+After the conversion process is finished, the command will create a `COVERAGE_REPORT_OUTPUT_FILE.json` file with the coverage report JSON readable by the CI integrations tool.
+
+# Dependencies
+
+> What is the project blocked on?
+
+This project has no blockers.
+
+> What will be impacted by the project?
+
+Impacts the publishing approach of the Coverage Converter tool.
+
+# Testing
+
+> How will the project be tested?
+
+This project will be tested manually. 
+
+# Alternatives Considered
+
+> Summarize alternative designs (pros & cons)
+
+# Coverage Converter publishing
+
+## Personal Package Archive (PPA), Homebrew, and Chocolatey.
+
+The alternative of the GitHub Releases is to publish the CLI to the package managers of different platforms. So, we'll deploy the CLI to the following package managers: 
+
+- `PPA` for the Linux users
+- `Homebrew` for the macOS users
+- `Chocolatey` for Windows users
+
+This alternative is pretty complex in configuration, but it provides an easy way of installing the Coverage Converter tool using the single platform-unified command. So, let's consider the main Pros & Cons of this alternative: 
+
+### Pros & Cons
+
+Pros: 
+- Easy to download the latest release.
+- Makes the CLI global-accessible command after installation.
+
+Cons: 
+- Complex configuration process.
+- Requires a separate configuration file for each platform (package management tool).
+- Some package management tools do not support publishing the compiled executables, so we should compile it on the user's machine.
+- Bad integration with GitHub Actions.
```

Comment:
```suggestion
- No existing integration with GitHub Actions.
```
Or can we be a bit more specific about what "bad" means here given your understanding of the process?